runtime/src/main/java/org/corfudb/runtime/object/transactions/TransactionalContext.java

```diff
@@ -84,6 +85,7 @@ public static boolean isInTransaction() {
      * @return          The context which was added to the transaction stack.
      */
     public static AbstractTransactionalContext newContext(AbstractTransactionalContext context) {
+        log.trace("TX begin[{}]", Utils.toReadableID(context.transactionID));
```

Comment:
should just be log.trace("TX...", context);, as we should rely on the toString override for the context